csrc/mmdeploy/codebase/mmdet/object_detection.cpp

```diff
@@ -127,8 +127,12 @@ Result<Detections> ResizeBBox::GetBBoxes(const Value& prep_res, const Tensor& de
     scale_factor = {1.f, 1.f, 1.f, 1.f};
   }
 
-  float w_offset = 0.f;
-  float h_offset = 0.f;
+  float w_offset = 0;
```

Comment:
When initializing a float variable, it is recommended to use `0.f` instead of `0`.
You can refer to https://stackoverflow.com/questions/5199338/what-is-the-significance-of-0-0f-when-initializing-in-c